tiatoolbox/dataloader/wsireader.py

```diff
@@ -648,10 +648,21 @@ def save_tiles(
                     raise ValueError
             # Raise index error if desired pyramid level not embedded
             # in level_dimensions
-            except (IndexError, ValueError):
+            except IndexError:
                 level = 0
                 slide_dimension = self.info.level_dimensions[level]
                 rescale = np.int(rescale)
```

Comment:
Should there be a warning ?

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/commit/2a241bb866b040df0d1b5b72124128f22ae30d97